cg/store/base.py

```diff
@@ -59,6 +59,13 @@ def _get_join_case_sample_query(self) -> Query:
         """Return join case sample query."""
         return self._get_query(table=CaseSample).join(CaseSample.case).join(CaseSample.sample)
 
+    def _join_sample_and_case(self):
+        return (
+            self.session.query(Case)
+            .join(CaseSample, Case.id == CaseSample.case_id)
+            .join(Sample, CaseSample.sample_id == Sample.id)
+        )
```

Comment:
What is the difference between this method and the _get_join_case_sample_query starting four rows above?

Reply:
This one starts from `Case` and uses a more explicit syntax for joining the tables.
The tests fail when using the other join-query

Reply:
Weird, feels as if one should be sufficient.

Reply:
<img width="868" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/e4fd1526-7eb2-4ef9-9b6e-f6be5547495a">

Reply:
I agree, tracked it here https://github.com/Clinical-Genomics/cg/issues/3073.
Will leave it as is for now